tests/browser/test_tools.py

```
@@ -52,16 +58,52 @@ def test_tool_execution_in_session(persona: NottePersona, action: EmailReadActio
-    with notte.Session(headless=True) as session:
-        agent = notte.Agent(session=session, persona=persona, max_steps=15)
-        resp = agent.run(
-            task=(
-                "Go to console.notte.cc, login with the email signup email, verify the account. "
-                "Stop after the account is verified, i.e as soon as your are on the 'One more second' page."
-                "CRITICAL: do not fill the in the onboarding form, just stop after the account is verified"
-            ),
-            url="https://console.notte.cc",
-        )
-        assert resp.success, f"Failed to run agent: {resp.answer}"
+    log_dir = Path(os.getenv("SIGNUP_DEBUG_LOG_DIR", "/tmp/signup-debug"))
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/signup-debug"

(S108)